### DISCUSSION TITLE
The problem with sensitive data.

### DISCUSSION THREAD

**Discussion by Kaczka212** [community/community#54425] [GitHub Education] [answered]
### Hi everyone,

Hello, I have some problems. I'm from Poland and I'm having trouble verifying my school ID for the github student program.

1. Everything is written in Polish.
2. I don't want to share sensitive data such as my face and "PESEL," which is an electronic system of citizen identification. The problem is that part of the school name overlaps with the "PESEL" number.
3. The validity date is on the back of the ID.

**Comment by Kaczka212** [comment]
![image](https://user-images.githubusercontent.com/80482933/236337920-61998ed1-76b3-475c-afb6-1542596a9134.png)
and that's another reason why I was rejected

**Reply by Kaczka212** [reply, marked as answer]
Okay, I made it.
There is a simple solution to these errors, just set your first and last name as the name for a moment and write something about the school in the bio

**Reply by jkazimierczak** [reply]
Did you sent just a back of student ID? If you also sent a front-side, did you remove (draw a rectangle over it/blur it/etc.) the PESEL and your photo?

**Comment by Kaczka212** [comment]
> Did you sent just a back of student ID? If you also sent a front-side, did you remove (draw a rectangle over it/blur it/etc.) the PESEL and your photo?

I only sent the school ID, and on the PESEL I placed a white card covering it. I also changed the name on GitHub to my first and last name for the duration of the verification.
